src/DemoRouter/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        var rc = new MessageRouterConfiguration();
+
+        rc.AddChannel(new MsmqTransport())
+            .HasEndpoint("Sales")//.AtMachine("ServerA")
+            .HasEndpoint("Finance");//.AtMachine("ServerB");
+
+        // Note to Kyle & Travis, the above code doesn't work yet. The `AtMachine` I just made up.
+        // Would it be possible to only have AtMachine available when you're on MsqmTransport?
```

Comment:
Yes, we can make AddChannel return an interface and then return a specific MSMQ config type that would have AtMachine

Reply:
@dvdstelt ☝️